### PR TITLE
Jitsi Update -> stable-5963

### PR DESCRIPTION
**Pull request by sakkiii**

### CODE REVIEW

**Comment by skepticalwaves** [comment]
Can this be made optional instead of being enabled by default when jitsi is enabled?

**Comment by sakkiii** [comment]
Jibri recording issues fixed in stable-5870 but here it's still didn't worked. :disappointed:

**Comment by sakkiii** [comment]
To make Jibri work need 80:80 and/or 443:443 port mapping, need lot more work. Any suggest or contribution are welcome.

@spantaleev I think we should test and review this PR as jitsi docker latest update stable-5963 and create PR for recording.

File: roles/matrix-jitsi/templates/prosody/env.j2 [hunk 1]
```diff
@@ -18,32 +18,35 @@ LDAP_TLS_CHECK_PEER={{ 1 if matrix_jitsi_ldap_tls_check_peer else 0 }}
...
-
 XMPP_MODULES={{ matrix_jitsi_xmpp_modules }}
 XMPP_MUC_MODULES=
 XMPP_INTERNAL_MUC_MODULES=
-
 XMPP_RECORDER_DOMAIN={{ matrix_jitsi_recorder_domain }}
-
-JICOFO_COMPONENT_SECRET={{ matrix_jitsi_jicofo_component_secret }}
+XMPP_CROSS_DOMAIN=true
+JICOFO_COMPONENT_SECRET
```
Comment (spantaleev) [inline roles/matrix-jitsi/templates/prosody/env.j2:31]: Why empty? What's happened to the jicofo component secret? Intentionally setting it to empty?

Comment (sakkiii) [reply roles/matrix-jitsi/templates/prosody/env.j2:31]: JICOFO_COMPONENT_SECRET removed in stable-5765 -> https://github.com/jitsi/docker-jitsi-meet/commit/953a4d2425df7128c35c4b1edebb88e8bfe28a97

Comment (spantaleev) [reply roles/matrix-jitsi/templates/prosody/env.j2:31]: Why not remove the reference completely?

Comment (sakkiii) [reply roles/matrix-jitsi/templates/prosody/env.j2]: yeah right done

**Comment by spantaleev** [comment]
I'm not using Jitsi anymore, so I can't easily test this, but it looks good. Thank you for updating Jitsi for us! :+1:

**Pull request merged**